client/payment-details/dispute-details/dispute-resolution-footer.tsx

```diff
@@ -0,0 +1,285 @@
+										'woocommerce-payments'
+								  ),
+							submissionDateFormatted
+						),
+						{
+							a: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content -- Link content is provided by createInterpolateElement
+								<a
+									target="_blank"
+									rel="noopener noreferrer"
+									href="https://woocommerce.com/document/woopayments/fraud-and-disputes/"
+								/>
+							),
+						}
+					) }
+				</FlexItem>
+				<FlexItem className="transaction-details-dispute-footer__actions">
+					<Button
+						variant="secondary"
+						onClick={ () => {
+							wcpayTracks.recordEvent(
+								wcpayTracks.events
+									.PAYMENT_DETAILS_VIEW_DISPUTE_EVIDENCE_BUTTON_CLICK,
+								{
+									dispute_status: dispute.status,
+								}
+							);
+							const challengeUrl = getAdminUrl( {
+								page: 'wc-admin',
+								path: '/payments/disputes/challenge',
+								id: dispute?.id,
+							} );
+							getHistory().push( challengeUrl );
+						} }
+					>
+						{ __(
+							'View submitted evidence',
+							'woocommerce-payments'
+						) }
+					</Button>
+				</FlexItem>
+			</Flex>
+		</CardFooter>
+	);
+};
+
+const DisputeWonFooter: React.FC< {
+	dispute: Dispute;
+} > = ( { dispute } ) => {
+	const closedDateFormatted = dispute?.metadata.__dispute_closed_at
+		? dateI18n(
+				'M j, Y',
+				moment
+					.unix(
+						parseInt( dispute.metadata.__dispute_closed_at, 10 )
+					)
+					.toISOString()
+		  )
+		: '-';
+
+	return (
+		<CardFooter className="transaction-details-dispute-footer">
+			<Flex justify="space-between">
+				<FlexItem>
+					{ createInterpolateElement(
+						sprintf(
+							/* Translators: %s - formatted date, <a> - link to documentation page */
+							__(
+								'Good news! You won this dispute on %s. The disputed amount and the dispute fee have been credited back to your account. <a>Learn more about preventing disputes</a>.',
+								'woocommerce-payments'
+							),
+							closedDateFormatted
+						),
+						{
+							a: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content -- Link content is provided by createInterpolateElement
+								<a
+									target="_blank"
+									rel="noopener noreferrer"
+									href="https://woocommerce.com/document/woopayments/fraud-and-disputes/"
+								/>
+							),
+						}
+					) }
+				</FlexItem>
+				<FlexItem className="transaction-details-dispute-footer__actions">
+					<Button
+						variant="secondary"
+						onClick={ () => {
+							wcpayTracks.recordEvent(
+								wcpayTracks.events
+									.PAYMENT_DETAILS_VIEW_DISPUTE_EVIDENCE_BUTTON_CLICK,
+								{
+									dispute_status: dispute.status,
+								}
+							);
+							const challengeUrl = getAdminUrl( {
+								page: 'wc-admin',
+								path: '/payments/disputes/challenge',
+								id: dispute?.id,
+							} );
+							getHistory().push( challengeUrl );
```

Comment:
Hiding the url/href in the `onClick` handler impacts usability in 2 ways:

- Can't cmd-click to open in new tab (moderate)
- Can't hover over button to see URL (minor)

However I can see that using `History::push` is much faster and nicer experience (is there a way we can have that AND retain normal browser link handling?). I'd like to get aligned on how we use history API and "soft" page loads in WooPayments and Woo admin generally. Personally I like using regular links as much as possible, so the user has more control (e.g. can open in separate tabs). 

Not a blocker, just sharing my personal opinion out loud - in my testing I've been frustrated when various Woo admin links don't let me open a new tab!

Reply:
I totally agree, thanks for pointing this out. I've addresses this in fa712764be75c8af65965dbceef8e9a82fc85dc7, let me know what you think!

As a workaround (solution), I've wrapped each `Button` with `Link` from `@woocommerce/components`, which [already handles the `getHistory().push()` functionality while allowing the CMD-click for native](https://github.com/woocommerce/woocommerce/blob/c661186499b30de3f232eee5bf1bfa94a01f90e0/packages/js/components/src/link/index.tsx#L40C7-L47).

Using `Button` without the `href`/`onClick` provides the correct UI.

Reply:
Sounds good to me! So is this a pattern you recommend in general? e.g. `<Link><Button /></Link>`?

Tested one of the buttons and it's fast and flexible 🙌🏻

Reply:
I found a little caveat with this approach – CMD+click won't run the `onClick` handler provided to `Link`,  therefore, tracks event recording should be conducted within the `Button`'s `onClick`.

Updated in e6e38ea973a9be94310320aec77af19311a6d244

Reply:
Good catch!